monitor/plugins/base.py

```diff
@@ -67,4 +67,3 @@ def run(self):
             except Exception as ex:
                 self.attempts -= 1
                 print ex.message
-                pass
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR?

Reply:
This is a redundant behavior.